rules/windows/defense_evasion_disabling_windows_logs.toml

```diff
@@ -31,7 +31,7 @@ process where event.type in ("start", "process_started") and
 
   ((process.name : ("pwsh.exe", "powershell.exe", "powershell_ise.exe") or process.pe.original_file_name in
       ("pwsh.exe", "powershell.exe", "powershell_ise.exe")) and
-	process.args : "Set-Service" and process.args: "EventLog" and process.args : "Disabled")  or
+	process.args : "Set-Service" and process.args: "EventLog" and process.args: "Limit-EventLog" and process.args: "OverwriteOlder" and process.args : "Disabled")  or
```

Comment:
Hey Austin 👋, I don't think this fits well within this rule. This would be a manipulation to logging configurations and possibly event overwrites, but it does not target a disabling of EventLog

Reply:
I was doing some reading and I came across this and thought it would be great to include in a rule.

I didn't think this would deserve it's own rule, but I do believe it deserves to be in a rule and we would have a missing component in the log rules if `Limit-Eventlog` was excluded. So you think this as a separate rule?

Reply:
I also don't think this one fits in this rule, I think an dedicated rule to windows event log tampering would be better

Reply:
Okay. I will work on separate rule logic.

Reply:
Nice, closing this one